src/App.tsx

```diff
@@ -74,27 +74,19 @@ const App = () => {
   // Create client
   useEffect(() => {
     const getClient = async () => {
-      try {
-        setClientIsLoading(true)
-        // Get clients
-        const clientResp = await createClient(settings)
-        setClient(clientResp)
-
-        console.log('Clients initialized.')
-
-        setSnackbarMessage({
-          text: `Connected to Alephium's Node "${settings.nodeHost}"!`,
-          type: 'info',
-          duration: 4000
-        })
-        setClientIsLoading(false)
-      } catch (e) {
-        setSnackbarMessage({
-          text: 'Unable to initialize the client, please check your network settings.',
-          type: 'alert'
-        })
-        setClientIsLoading(false)
-      }
+      setClientIsLoading(true)
+      // Get clients
+      const clientResp = await createClient(settings)
+      setClient(clientResp)
+
+      console.log('Clients initialized.')
```

Comment:
Why plural, since it's one client?

Reply:
The main backend "client" is actually an aggregation of the cliqueClient (node) and the ExplorerClient (explorer-backend).